### PR TITLE
ci: cover more features on Linux and Windows

### PR DESCRIPTION
**Pull request by kmilos**
See https://github.com/darktable-org/darktable/pull/10463

Windows changes seem substantial, but it's only portmidi and flickcurl, the rest is due to sorting...

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -379,38 +381,40 @@ jobs:
...
             mingw-w64-x86_64-ninja
-            mingw-w64-x86_64-libxml2
-            mingw-w64-x86_64-pugixml
-            mingw-w64-x86_64-libjpeg-turbo
-            mingw-w64-x86_64-zlib
+            mingw-w64-x86_64-nsis
+            mingw-w64-x86_64-dbus-glib
+            mingw-w64-x86_64-drmingw
             mingw-w64-x86_64-exiv2
+            mingw-w64-x86_64-flickcurl
```
Comment (TurboGit) [inline .github/workflows/ci.yml:388]: can be removed as it is not used anymore. we do not support flickr or any other external library service now except the Open Source Piwigo.

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -41,38 +41,40 @@ jobs:
...
             mingw-w64-x86_64-ninja
-            mingw-w64-x86_64-libxml2
-            mingw-w64-x86_64-pugixml
-            mingw-w64-x86_64-libjpeg-turbo
-            mingw-w64-x86_64-zlib
+            mingw-w64-x86_64-nsis
+            mingw-w64-x86_64-dbus-glib
+            mingw-w64-x86_64-drmingw
             mingw-w64-x86_64-exiv2
+            mingw-w64-x86_64-flickcurl
```
Comment (TurboGit) [inline .github/workflows/nightly.yml:48]: likewise

**Review by TurboGit — CHANGES_REQUESTED** [review]
Minor comment apart from that it looks ok to me.

**Comment by TurboGit** [comment]
And so we can remove libflickcurl-dev from Linux script and Dockerfile.

File: .github/workflows/nightly.yml (hunk 2 above)
Comment (TurboGit) [reply .github/workflows/nightly.yml:48]: See 39a47f6aa1780fb1fcb6f95130802acadbdfbe59 for why it has been removed.

**Comment by kmilos** [comment]
So, remove flickurl from Linux builds too?

**Comment by kmilos** [comment]
Ok, removed flickcurl from CI and instructions

**Review by TurboGit — APPROVED** [review]
Looks good, thanks!

**Pull request merged**